tests/e2e/utils/utils.go

```diff
@@ -134,31 +134,29 @@ func RunSQLTest(ctx context.Context, frontendIngressIP string, isDistributed boo
 	}
 
 	const (
-		createDistributedTableSQL = `CREATE TABLE dist_table(
+		createDistributedTableSQL = `CREATE TABLE dist_table (
+                             ts TIMESTAMP DEFAULT current_timestamp(),
+                             n INT,
+    					     row_id INT,
+                             TIME INDEX (ts),
+                             PRIMARY KEY(n)
+                        )
+                        PARTITION ON COLUMNS (n) (
+                          n < 5,
+                          n >= 5 AND n < 9,
+                          n >= 9
+					    )`
+
+		createStandaloneTableSQL = `CREATE TABLE dist_table (
 							ts TIMESTAMP DEFAULT current_timestamp(),
-							n INT,
-							row_id INT,
-							PRIMARY KEY(n),
-							TIME INDEX (ts)
-						  )
-                          PARTITION ON COLUMNS (n) (
-						      n < 5,
-							  n >= 5 AND n < 9,
-							  n >= 9
-						  )
-						  engine=mito;`
-
-		createStandaloneTableSQL = `CREATE TABLE dist_table(
-							ts TIMESTAMP DEFAULT current_timestamp(),
-							n INT,
-							row_id INT,
-							PRIMARY KEY(n),
-							TIME INDEX (ts)
-						  )
-						  engine=mito;`
+                            n INT,
+    					    row_id INT,
+                            TIME INDEX (ts),
+                            PRIMARY KEY(n)
+						  )`
 
-		insertDataSQL = `INSERT INTO dist_table(n, row_id) VALUES (?, ?);`
-		selectDataSQL = `SELECT * FROM dist_table`
+		insertDataSQLStr = `INSERT INTO dist_table(n, row_id) VALUES (%d, %d)`
```

Comment:
why changes `insertDataSQL`? I think use `?` which is supported by mysql driver will be better.

Reply:
I try to use GreptimeDB image tag [v0.7.2](https://github.com/GreptimeTeam/greptimedb-operator/pull/154/commits/4e4eb753f25d8ebb2b377f72e3671f2295dfaf0b) is successfully in e2e test.

Reply:
Seems it's a incompatible of db. According to the error, you have to specify the `ts` column in insert statement.

I fix it in PR #157 with other e2e fix. After #157 merged, you can rebase the code.